patches.def.h

```diff
@@ -155,6 +155,11 @@
 /* Show tag symbols in the bar. */
 #define BAR_TAGS_PATCH 1
 
+/* Allow configuration of colors for tags. Looks good with the underline patch.
+ * https://github.com/fitrh/dwm/issues/1
+ */
+#define BAR_COLORFULTAGS_PATCH 0
+
```

Comment:
This should be moved out of the bar module section and into the bar options section, probably right beneath BAR_COLOR_EMOJI_PATCH.

Reply:
I think it makes sense to place it under `BAR_TAGS_PATCH` as it would be easier to navigate.